### PR TITLE
feat: add Wordpress runtime version reporting

### PR DESCRIPTION
**Pull request by tomlongridge**
Uses new hook in Diagnostics to append the Wordpress application version to device data to
make it reportable.

## Goal

Augments reporting with the version of Wordpress in use when an event occurred in case of issues in a specific version of the framework.

## Design

Added data to device.runtimeVersions.wordpress as per other notifiers.

### Changed

* bugsnag.constructBugsnag - appends the runtime version information to the device data through the Bugsnag client

## Tests

* No unit tests present in repo - checked the data was present after running the example Wordpress project (requires the feature branch of bugsnag-php **v2**)

## Review

For the submitter, initial self-review:

- [x] Commented on code changes inline explain the reasoning behind the approach
- [x] Reviewed the test cases added for completeness and possible points for discussion
- [x] A changelog entry was added for the goal of this pull request
- [x] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [ ] Initial review of the intended approach, not yet feature complete
  - [ ] Structural review of the classes, functions, and properties modified
  - [x] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [x] Consistency across platforms for structures or concepts added or modified
- [x] Consistency between the changeset and the goal stated above
- [x] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [x] Usage friction - is the proposed change in usage cumbersome or complicated?
- [x] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [x] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [x] Thoroughness of added tests and any missing edge cases
- [x] Idiomatic use of the language

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+## TBD
+
+* Add WordPress version string to report and session payloads (device.runtimeVersions)
```
Comment (mikeewhite) [inline CHANGELOG.md]: The Wordpress integration doesn't currently support session tracking.
